src/sentry/models/organizationmember.py

```diff
@@ -252,6 +252,15 @@ def regenerate_token(self):
         self.token = self.generate_token()
         self.refresh_expires_at()
 
+    def outbox_for_create(self) -> RegionOutbox:
+        return RegionOutbox(
+            shard_scope=OutboxScope.ORGANIZATION_SCOPE,
+            shard_identifier=self.organization_id,
+            category=OutboxCategory.ORGANIZATION_MEMBER_CREATE,
```

Comment:
Should we use `ORGANIZATION_MEMBER_UPDATE` here? With a separate create category, we could encounter a situation where an organization member update message is processed before the create (as they are separate categories).